### DISCUSSION TITLE
Issue with PostgreSQL 16

### DISCUSSION THREAD

**Discussion by RaimundasZutautas** [benjaminjonard/koillection#925] [Q&A] [answered]
Hello,
Today Watchtower automatically updated my PostgreSQL to v16 and my koillectionDB cannot start.
It showing in logs that version is incompatible:
"2023-09-16 19:13:00.619 UTC [1] FATAL:  database files are incompatible with server
2023-09-16 19:13:00.619 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 15, which is not compatible with this version 16.0 (Debian 16.0-1.pgdg120+1)."

Maybe someone knows how to fix it?
My koillection running on Synology NAS Docker.

**Comment by benjaminjonard** [comment, marked as answer]
Hello, 

PostgreSQL major versions are not compatible with each other and always require manual upgrade of the database. The doc can be found here: https://www.postgresql.org/docs/current/upgrading.html

If you plan to do this, I strongly suggest making a backup of your database first.

Alternatively, you can update the docker-compose file and specify version 15 for PostgreSQL.
